app/models/mixins/reportable_mixin.rb

```diff
@@ -93,23 +93,17 @@ def search(count = :all, options = {})
 
     # private
 
-    def get_include_for_find(report_option)
-      includes = report_option
-      if includes.kind_of?(Hash)
-        result = {}
-        includes.each do |k, v|
+    def get_include_for_find(includes)
+      case includes
+      when Hash
+        includes.each_with_object({}) do |(k, v), result|
           v[:include] = v["include"] if v["include"]
```

Comment:
Yea. Looks like `get_include_for_find` is used in many places. I'm kind of afraid of changing observable behaviours of such methods since I'm not very familiar with the app - who knows what might break.